package.json

```diff
@@ -83,6 +83,7 @@
     "chromedriver": "^2.46.0",
     "clean-webpack-plugin": "^0.1.19",
     "copy-webpack-plugin": "^4.5.2",
+    "core-js": "^2.5.7",
```

Comment:
I saw that v2.6.9 is the latest v2.x release, any reason to not `^2.6.9`?

Reply:
Good question, I was following the advice of a comment in a SO question a little too close so it was probably the latest at that time, but arbitrary for us now. I just tried ^2.6.9 and that will work well.